application/account-management/Core/Integrations/Gravatar/GravatarClient.cs

```diff
@@ -0,0 +1,54 @@
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+using PlatformPlatform.AccountManagement.Features.Users.Avatars;
+using PlatformPlatform.AccountManagement.Features.Users.Domain;
+using PlatformPlatform.AccountManagement.TelemetryEvents;
+using PlatformPlatform.SharedKernel.TelemetryEvents;
+
+namespace PlatformPlatform.AccountManagement.Integrations.Gravatar;
+
+public sealed class GravatarClient(
```

Comment:
Suggestion: Given our discussion last week on what a Client is, this should be called something different. Maybe ...drumroll... Service?

(We agreed that Clients only contain integration code, and thus make no internal calls, as is done here. Makes a lot of sense when you e.g. see GravatarClient, and a convention many API providers use in their SDKs. 

Furthermore, if Clients are only integrations, they provide a really nice abstraction for any external infrastructure. And by being thin, they are trivial to mock, thus leaving the system easily testable.)